includes/yarpp_fulltext_notice.php

```diff
@@ -31,8 +31,15 @@
     }
 }
 
-$table_type = $yarpp->diagnostic_myisam_posts();
-if ((bool) $table_type !== true) $yarpp->disable_fulltext();
+$fulltext_posts = $yarpp->diagnostic_fulltext_posts();
+if ($fulltext_posts === true) {
+    if ((bool) get_option('yarpp_fulltext_disabled', false) === true) {
+        $yarpp->enable_fulltext();
+        update_option('yarpp_fulltext_disabled', 0);
```

Comment:
Hi I'm working on integrating this into YARPP on wp.org. Can you explain why calling `$yarpp->enable_fulltext();` was necessary here?